packages/plugins/rubygem-newt/rubygem-newt.spec

```diff
@@ -1,101 +1,77 @@
-# template: scl
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+# template: default
 %global gem_name newt
 %global gem_require_name %{gem_name}
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.9.7
-Release: 3%{?dist}
+Name: rubygem-%{gem_name}
+Version: 1.0.1
+Release: 1%{?dist}
 Summary: Ruby bindings for newt
-Group: Development/Languages
 License: MIT
 URL: https://github.com/theforeman/ruby-newt
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
 # start specfile generated dependencies
-Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
-BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}ruby-devel
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
-Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
-# end specfile generated dependencies
-
-Requires: newt
+Requires: ruby
+BuildRequires: ruby-devel
 BuildRequires: newt-devel
```

Comment:
This should be outside the `generated dependencies` block so it won't be overwritten the next time the scripts run.

Reply:
I'm not sure If I understand. So, do I need to move it to a different block? Where?

Reply:
Oh yeah I see it now.